unit_tests/test_xunit.py

```diff
@@ -101,6 +113,39 @@ class DummyStream:
         f.close()
         return data
 
+class TestXMLOutputWithXMLAndPrefix(BaseTestXMLOutputWithXML):
+    def setUp(self):
+        super(TestXMLOutputWithXMLAndPrefix, self).setUp()
+        self.configure([
+            "--with-xunit",
+            "--xunit-file=%s" % self.xmlfile,
+            "--xunit-prefix-with-testsuite-name"
```

Comment:
This all looks good with one exception: I expected to see a test that uses xunit-testsuite-name to change the prefix.  Do you mind adding such a test?